oio/xcute/orchestrator.py

```diff
@@ -204,12 +204,13 @@ def handle_job(self, job_info):
                 job_info['job']['id'], job_info['job']['type'], exc)
             self.backend.fail_job(job_info['job']['id'])
 
-    def _beanstalkd_job_data_from_task(self, job, task_class, item, kwargs):
+    def _beanstalkd_job_data_from_task(self, job, task_class, task_item,
+                                       task_kwargs):
         beanstalkd_job = dict()
         beanstalkd_job['job_id'] = job.job_id
         beanstalkd_job['task'] = pickle.dumps(task_class)
-        beanstalkd_job['item'] = item
-        beanstalkd_job['kwargs'] = kwargs or dict()
+        beanstalkd_job['item'] = task_item
```

Comment:
what is the difference between an item and kwargs?
kwargs is also not a good name, that's just a python thing